backend/src/api.ts

```diff
@@ -261,6 +280,9 @@ loginCheckedPost('/updateCandidateDeciderRating', (req, user) =>
 loginCheckedPost('/updateCandidateDeciderComment', (req, user) =>
   updateCandidateDeciderComment(user, req.body.uuid, req.body.id, req.body.comment).then(() => ({}))
 );
+loginCheckedPost('/sendMail', async (req, user) => ({
+  info: await sendMail(req.body.to, req.body.subject, req.body.text)
+}));
```

Comment:
Should we add a comment that this just for testing at the moment?

Reply:
Alright thanks for explaining, I got confused reading through the conversations before when you guys were talking about leaving a testing endpoint in.

Reply:
yeah no problem -- i initially didn't think that we needed it but in implementing it later on (after the semester ended) this was the only way that I could get it to work